### PR TITLE
Fix broken links in Training Operator #3812

### PR DESCRIPTION
**Pull request by varodrig**
Fix broken links in Training Operator #3812

### CODE REVIEW

**Comment by varodrig** [comment]
Need confirmation on these:

1-  Kuberentes Api Version,  I'm looking for confirmation on the version to be used. Right now I used the latest I found in other docs. 1.30.
image

 2- I noticed that other files are using Api version 1.12, 1.30 and 1.18.  Should I create an issue to address this difference?
Pasted Graphic 7

3- I couldn't find the right links to be replaced in the tensorflow doc. please advise.

See the guide to [logging and monitoring](/docs/gke/monitoring/) for
instructions on getting logs using Stackdriver.

As described in the guide to
[logging and monitoring](https://www.kubeflow.org/docs/gke/monitoring/#filter-with-labels),

@hbelmiro

**Comment by hbelmiro** [comment]
> 1- Kuberentes Api Version, I'm looking for confirmation on the version to be used. Right now I used the latest I found in other docs. 1.30.

I think you should use 1.27, which is the [minimal requirement](https://www.kubeflow.org/docs/components/training/installation/#prerequisites).

> 2- I noticed that other files are using Api version 1.12, 1.30 and 1.18. Should I create an issue to address this difference?

Yes, please.

> 3- I couldn't find the right links to be replaced in the tensorflow doc. please advise.
> 
> See the guide to [logging and monitoring](/docs/gke/monitoring/) for instructions on getting logs using Stackdriver.
> 
> As described in the guide to [logging and monitoring](https://www.kubeflow.org/docs/gke/monitoring/#filter-with-labels),
> 
> 4- Replaced missing job, confirm you are ok with this new value image

@andreyvelich, do you know the answers for 3 and 4?

File: content/en/docs/components/katib/user-guides/trial-template.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ In Katib examples, you can find the following examples for Trial's Workers:
 
 - [Kubeflow `PyTorchJob`](/docs/components/training/user-guides/pytorch/)
 
-- [Kubeflow `MXJob`](/docs/components/training/user-guides/mxnet)
+- [Kubeflow `PaddleJob`](/docs/components/training/user-guides/paddle)
```
Comment (andreyvelich) [inline content/en/docs/components/katib/user-guides/trial-template.md:23]: We don't have Paddle example in Katib for now: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/kubeflow-training-operator.
Maybe you can just remove the MXJob from this list.

**Review by andreyvelich — COMMENTED** [review]
Thank you for doing this @varodrig!
/assign @kubeflow/wg-training-leads

**Comment by andreyvelich** [comment]
>3- I couldn't find the right links to be replaced in the tensorflow doc. please advise.


I think, this guide has been removed a while ago. Let's remove it from the docs.

**Comment by varodrig** [comment]
@hbelmiro  I'll update the api version to 1.29 as discussed on the slack.

**Review by hbelmiro — COMMENTED** [review]
/lgtm
/ok-to-test

**Review by tenzen-y — COMMENTED** [review]
Thanks!
LGTM

**Review by andreyvelich — COMMENTED** [review]
Thanks @varodrig!
/lgtm
/approve

**Pull request merged**